### PR TITLE
Overall formatting

### PR DESCRIPTION
**Pull request by nicktate**
@normanjoyner @jeremykross

### CODE REVIEW

File: lib/kubernetes-api.js [hunk 1]
```diff
@@ -62,95 +62,90 @@ class KubernetesApi extends ApiImplementation {
...
+            uri: `${this.k8s_api_url}/api/v1/namespaces/default/services`,
             method: 'POST',
             json: true,
             body: service_body
         };
 
         return request(req, (err, res) => {
             if(err || res.statusCode !== 201) {
-                return errorHandler(`Could not create service: ${service_body.metadata.name} for application: ${app_name}`);
+                return cb(`Could not create service: ${service_body.metadata.name} for application: ${app_name}`);
```
Comment (normanjoyner) [inline lib/kubernetes-api.js]: can we return all of these as `Error`?

File: lib/kubernetes-api.js [hunk 2]
```diff
@@ -62,95 +62,90 @@ class KubernetesApi extends ApiImplementation {
...
-        const appName = _.get(appDesc, ['metadata', 'name']);
+    static k8sServiceDescriptionFor(app_desc, ports, discovery_ports) {
+        const app_name = _.get(app_desc, ['metadata', 'name']);
 
-        return{
+        return {
             kind: 'Service',
             metadata: {
-                name: `ser${_.join(_.take(String(encode(appName)), 19), '')}` // Limit to 24 chars
+                name: `ser${_.join(_.take(String(encode(app_name)), 19), '')}` // Limit to 24 chars
```
Comment (normanjoyner) [inline lib/kubernetes-api.js]: update comment

File: lib/kubernetes-api.js [hunk 3]
```diff
@@ -62,95 +62,90 @@ class KubernetesApi extends ApiImplementation {
...
-                this.updateApplication(app.id, {}, {
-                    errorHandler: (err) => {
-                        // eslint-disable-next-line no-console
-                        console.log(`Error updating application when enforcing constraints for ${app.id}: ${err}`);
-                    },
-                    handler: () => {
-                        // eslint-disable-next-line no-console
-                        console.log(`Enforced constraints for ${app.id}`);
+        return this.getApplications((err, apps) => {
+            if (err) {
```
Comment (normanjoyner) [inline lib/kubernetes-api.js]: remove space after `if`

File: lib/kubernetes-api.js [hunk 4]
```diff
@@ -227,137 +223,127 @@ class KubernetesApi extends ApiImplementation {
...
-                    return errorHandler('Failed to retrive k8s pods inside getContainers call');
+                    return cb('Failed to retrive k8s pods inside getContainers call');
                 }
 
                 const pods = util.safeParse(body).items;
 
                 const containers = _.map(pods, pod => {
-                    const nodeName = _.get(pod, 'spec.nodeName');
-                    const k8sContainer = _.get(pod, 'spec');
+                    const node_name = _.get(pod, 'spec.nodeName');
```
Comment (normanjoyner) [inline lib/kubernetes-api.js]: array

File: lib/kubernetes-api.js [hunk 5]
```diff
@@ -227,137 +223,127 @@ class KubernetesApi extends ApiImplementation {
...
+                    const k8s_container = pod.spec;
 
-                    const status = Translator.csStatusFromK8SStatus(_.get(pod, 'status'));
+                    const status = Translator.csStatusFromK8SStatus(pod.status);
 
                     const start_time = Date.parse(_.get(pod, 'status.startTime'));
-                    const containerID = _.get(pod, 'metadata.uid');
+                    const container_id = _.get(pod, 'metadata.uid');
```
Comment (normanjoyner) [inline lib/kubernetes-api.js]: array

**Comment by nicktate** [comment]
@normanjoyner updated

File: lib/kubernetes-api.js (hunk 4 above)
Comment (nicktate) [inline lib/kubernetes-api.js]: converted all to dot notation

File: lib/kubernetes-api.js (hunk 5 above)
Comment (nicktate) [inline lib/kubernetes-api.js]: converted all to dot notation

**Pull request merged**